### DISCUSSION TITLE
How to mount volume with z option by default

### DISCUSSION THREAD

**Discussion by akizminet** [containers/podman#16582] [Q&A]
I am working with a project use docker to build code. All `docker run` commands in project don't mount volume with `:z` or `:Z` suffix. Therefore I get many permission denied (on Fedora) when run automation job with `podman-docker`. Does podman have any options to deal with this problem?

**Comment by rhatdan** [comment]
Could you give us an example of a podman command that is blowing up.

You could label the volumes by default with container_file_t:s0 and then they would all be shareable.

BTW Docker would have the same problem if you ran it with SELinux enabled.
